client/src/Component/EventBodyComponent/EventBodyComponent.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import AppBar from "@material-ui/core/AppBar";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import Typography from "@material-ui/core/Typography";
+import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
```

Comment:
These can be combined as `import { AppBar, Tabs, etc. } from "@material-ui/core";`